pkg/cmd/cli/cmd/edit.go

```diff
@@ -23,14 +22,27 @@ import (
 	"github.com/openshift/origin/pkg/util/jsonmerge"
 )
 
+type EditOptions struct {
```

Comment:
@rhcarvalho I know about that and @kargakis as well ;-) And while I don't have time fixing all of them I also don't want to introduce new ones to fix later...

We just have to wait for the Santa that runs golint and fix all the missing comments ;-)